tls/_common/enums.py

```diff
@@ -45,6 +45,8 @@ class HandshakeType(Enum):
     CERTIFICATE_VERIFY = 15
     CLIENT_KEY_EXCHANGE = 16
     FINISHED = 20
+    CERTIFICATE_URL = 21
+    CERTIFICATE_STATUS = 22
```

Comment:
Do you think a later PR can make the `Handshake` construct use `EnumSwitch` to dispatch to the various handshake types?